tools/ci/truffle_test

```diff
@@ -2,12 +2,17 @@
 
 set -e
 
-yarn solhint "evm/**/*.sol"
-yarn solhint "examples/**/*.sol"
+yarn workspace chainlink run solhint
+yarn workspace chainlinkv0.5 run solhint
+yarn workspace @chainlink/box run solhint
+yarn workspace @chainlink/example-echo-server run solhint
+yarn workspace @chainlink/example-testnet run solhint
+yarn workspace @chainlink/example-twilio-sms run solhint
+yarn workspace @chainlink/example-uptime-sla run solhint
```

Comment:
Should we have all of these workspace commands above contain a `lint` key which runs `solhint`? Then we could just run `yarn lint:all` or something in the root package instead of duplicating logic here.

Reply:
Not all packages use solidity. It feels a bit weird to add `solhint` no-op to OperatorUI for example.